python/lsst/pipe/tasks/cosmicRayPostDiff.py

```diff
@@ -0,0 +1,263 @@
+    )
+    refObjLoader = pexConfig.ConfigurableField(
+        target=LoadIndexedReferenceObjectsTask,
+        doc="reference object loader",
+    )
+    repairPostDiff = pexConfig.ConfigurableField(
+        target=RepairPostDiffTask,
+        doc="Remove cosmic rays",
+    )
+    checkUnitsParseStrict = pexConfig.Field(
+        doc="Strictness of Astropy unit compatibility check, can be 'raise', 'warn' or 'silent'",
+        dtype=str,
+        default="raise",
+    )
+
+    def setDefaults(self):
+        super().setDefaults()
+
+# \addtogroup LSST_task_documentation
+# \{
+# \page CosmicRayPostDiffTask
+# \ref CosmicRayPostDiffTask_ "CosmicRayPostDiffTask"
+# \copybrief CosmicRayPostDiffTask
+# \}
+
+
+class CosmicRayPostDiffTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
+    r"""!Measure bright sources and use this to estimate background and PSF of an exposure
+
+    @anchor CosmicRayPostDiffTask_
+
+    @section pipe_tasks_cosmicRayPostDiff_Contents  Contents
+
+     - @ref pipe_tasks_cosmicRayPostDiff_Purpose
+     - @ref pipe_tasks_cosmicRayPostDiff_Initialize
+     - @ref pipe_tasks_cosmicRayPostDiff_IO
+     - @ref pipe_tasks_cosmicRayPostDiff_Config
+     - @ref pipe_tasks_cosmicRayPostDiff_Debug
+
+
+    @section pipe_tasks_cosmicRayPostDiff_Purpose  Description
+
+    Given an output image from image differencing:
+    - repair cosmic rays
+    - At the moment this task does NOT recompute the PSF or re-do source detection.
+
+    @section pipe_tasks_cosmicRayPostDiff_Initialize  Task initialisation
+
+    @copydoc \_\_init\_\_
+
+    @section pipe_tasks_cosmicRayPostDiff_IO  Invoking the Task
+
+    Call all the `run` method:
+
+    @section pipe_tasks_cosmicRayPostDiff_Config  Configuration parameters
+
+    See @ref CosmicRayPostDiffConfig
+
+    @section pipe_tasks_cosmicRayPostDiff_Debug  Debug variables
+
+    The @link lsst.pipe.base.cmdLineTask.CmdLineTask command line task@endlink interface supports a flag
+    `--debug` to import `debug.py` from your `$PYTHONPATH`; see @ref baseDebug for more about `debug.py`.
+
+    CosmicRayPostDiffTask has a debug dictionary with the following keys:
+    <dl>
+    <dt>frame
+    <dd>int: if specified, the frame of first debug image displayed (defaults to 1)
+    <dt>repair
+    <dd>bool; if True display image and sources after final repair
+
+    </dl>
+
+    For example, put something like:
+    @code{.py}
+        import lsstDebug
+        def DebugInfo(name):
+            di = lsstDebug.getInfo(name)  # N.b. lsstDebug.Info(name) would call us recursively
+            if name == "lsst.pipe.tasks.cosmicRayPostDiff":
+                di.display = dict(
+                    repair = True,
+                )
+
+            return di
+
+        lsstDebug.Info = DebugInfo
+    @endcode
+    into your `debug.py` file and run `calibrateTask.py` with the `--debug` flag.
+
+    Some subtasks may have their own debug variables; see individual Task documentation.
```

Comment:
This documentation is mostly irrelevant. I would delete it, rather than trying to fix it up.

Reply:
I got rid of most of it but left the first few lines and copydoc bits. All of the debug stuff is gone now.